sdk/policyinsights/Azure.ResourceManager.PolicyInsights/tests/Scenario/PolicyMetadataTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.PolicyInsights.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.PolicyInsights.Tests
+{
+    internal class PolicyMetadataTests : PolicyInsightsManagementTestBase
+    {
+        private PolicyMetadataCollection _metadataCollection;
+        public PolicyMetadataTests(bool isAsync) : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public void TestSetUp()
+        {
+            _metadataCollection = DefaultTenant.GetAllPolicyMetadata();
+        }
+
+        [RecordedTest]
+        [Ignore("Deserialize Error: System.UriFormatException: Invalid URI: The URI is empty.")]
```

Comment:
```json
{
  "properties": {
    ......
    "additionalContentUrl": "7"
  },
  "id": "/providers/Microsoft.PolicyInsights/policyMetadata/NZISM_Security_Benchmark_v1.1_SS-2",
  "name": "NZISM_Security_Benchmark_v1.1_SS-2",
  "type": "Microsoft.PolicyInsights/policyMetadata"
}
```
We have a rename rule in autorest.md that changes all '*Uri' types to Uri. There has a property `additionalContentUrl` in PolicyMetadata, and lots of its values will return string.empty, and a special item showed above, will return "7" therefore GetAll() method failed. How should we fix this

Reply:
If the value of this property is not always a real `Uri`, we should use custom code to add a string value property like `additionalContentUriString` and EB Nerver the existing `additionalContentUri`

Reply:
Resolved in [this commit](https://github.com/Azure/azure-sdk-for-net/pull/34797/commits/893d0f57cbcd01edb7afef4ad244eddf95976d5d), please review thanks